pyafq_cli.json

```diff
@@ -0,0 +1,501 @@
+{
+    "name": "pyAFQ",
+    "tool-version": "0.1.1",
+    "description": "Automated Fiber Quantification ... in Python",
+    "command-line": "pyAFQ [CONFIG_FILE]",
+    "schema-version": "0.5",
+    "container-image": {
```

Comment:
***ToDo @gkiar***: add `environment-variables` field here to make explicit the `fuzzy` parameters.